internal/pkg/diff/diff_test.go

```diff
@@ -409,12 +409,13 @@ func TestDiffRelations(t *testing.T) {
 	differ := NewDiffer(projectState)
 	reporter := differ.diffValues(objectState.Key(), rObject.Relations, lObject.Relations)
 	expected := `
-  - api side relation "path/to/target"
   - manifest side relation mocked key "foo"
-  + api side relation mocked key "002"
   + manifest side relation mocked key "bar"
+  - api side relation "path/to/target"
```

Comment:
V tomto teste sa to prejavi tak, ze najprv su vsetky rozdiely z `InManifest` a potom `InApi`.